.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:9.4
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      RAILS_ENV: test
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
+      - name: Configure Ruby env
+        run: |
+          echo "gem: --no-document" > ~/.gemrc
+          bundle config set clean 'true'
+      - name: Ruby gem cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Install gems
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - name: Check for untracked changes in Gemfile.lock
+        uses: ./.github/actions/check_untracked_changes
+        with:
+          path: "./Gemfile.lock"
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.16.0
+          cache: 'yarn'
+      - name: Find yarn cache location
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: JS package cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install packages
+        run: |
+          yarn install --pure-lockfile
+      - name: Setup Code Climate test-reporter
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build        
+      - name: Setup Database
+        run: |
+          cp .github/test_db/config/database.yml config/database.yml
```

Comment:
Mostly because the CircleCI integration currently uses the db configuration file ".circleci/database.yml" which it copies to replace "config/database.yml". We're basically doing the same the previous CI was doing.

I guess we can use config/database.yml.example if it works properly and if it's okay to depend on this example file.

Reply:
Can't we use env variables to configure the database and track `config/database.yml` here like suggested in [this comment](https://github.com/rootstrap/special-olympics-rails-api/pull/6#discussion_r852162701)? @JulianPasquale